web/app.php

```diff
@@ -27,12 +33,11 @@
 // See http://symfony.com/doc/current/book/http_cache.html#symfony-reverse-proxy
 //
 // require_once __DIR__.'/../app/AppCache.php';
-// $kernel = new AppKernel('prod', false);
-// $kernel->loadClassCache();
 // $kernel = new AppCache($kernel);
 // Request::enableHttpMethodParameterOverride();
```

Comment:
@javiereguiluz this is what I suggest changing actually. Making the site expect a delete request which would always be emulated using a way requiring a CSRF token to be safe is not worth it IMO. It still does not allow to perform a real delete request for supported clients (because you would still need the CSRF token) and browsers will all require using POST. I suggest acknowledging it fully and aviding the need to allow parameter-based method overriding